lib/js/src/rpc/RpcStruct.js

```diff
@@ -125,6 +125,21 @@ class RpcStruct {
         }
     }
 
+    /**
+     * 
+     * @param {Object} tClass - Class to validate with.
+     * @param {Array} obj - Array with instance tClass
+     */
+    validateArrayType (tClass, obj) {
+        if (!Array.isArray(obj)) {
```

Comment:
Recommend moving this logic into the `validateType` method and adding a third (optional) argument to the method definition: `isArray = false`.